app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/SelectSongScreen.kt

```diff
@@ -220,6 +220,12 @@ fun handleItemClick(
     viewModel.retrieveChild(listItem)
     return
   }
+  if (listItem.id.contains("spotify:") ||
+      listItem.id.isBlank()) { // TODO: create an issue for handling this
```

Comment:
Make sure to create this issue if you haven't already done so

Reply:
yes chef